### ISSUE TITLE
clarification in Readme.md

### ISSUE THREAD

**Issue by mi-hol** [vexx32/PSKoans#47]
Great project!

Trying to follow but hit some hurdles
1. Clone the Repo
currently it states:
```
git clone the repository into your desired directory, or download and extract the repository as a .zip file into a directory of your choice
```
guess it actually meant:

```
git clone the repository into your desired directory, or download the repository as a .zip file and extract  into a directory of your choice
```

2. run Measure-Karma fails for me, did I make an error?

```
PS C:\Users\user> Install-Module PSKoans -Scope CurrentUser
PS C:\Users\user> Measure-Karma
Measure-Karma : The term 'Measure-Karma' is not recognized as the name of a cmdlet, function, script file, or operable program.

Versions:
-  PSKoans 0.39.6.

-  $psversiontable

Name                           Value
----                           -----
PSVersion                      6.0.3
PSEdition                      Core
GitCommitId                    v6.0.3
OS                             Microsoft Windows 10.0.17134
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by vexx32** [comment]
yeah, I gotta fix that readme a bit, thank you!

as for the command, I guess I haven't pushed that to the psgallery just yet. if you want to use that, install the GitHub version of the module.

the gallery version is still using `Get-Enlightenment` I think... (which is a handful!)

you can also call it with `rake` in both versions. I'll have it pushed to the gallery within a few days time

**Comment by vexx32** [comment]
Fixed in #50 :)
